src/screens/EventScreens/CreateEventScreen/index.js

```diff
@@ -12,7 +17,16 @@ const CreateEventScreen = ({ navigation }) => {
   const [eventLocation, setEventLocation] = useState('');
   const [eventDescription, setEventDescription] = useState('');
   const [isCreateEnabled, setIsCreateEnabled] = useState(true);
-
+  const { userInfo } = useAuthStore();
+  const { createEvent } = useEventStore();
+  const eventInfoObj = {
```

Comment:
Put this object in the `onPressCreate ` function if it is only used in that function. Otherwise may cause unnecessary rerenders.